src/ralph/filters.py

```diff
@@ -0,0 +1,38 @@
+"""
+Ralph tracking logs filters.
+"""
+from .exceptions import EmptyEventsCollection, EventKeyError
+
+
+def empty(events):
+    """Check that an events collection is not empty.
+
+    Args:
+        events (DataFrame): events to filter
+
+    Returns:
+        Unfiltered input events (pandas DataFrame).
+
+    """
+
+    if len(events) < 1:
+        raise EmptyEventsCollection("Input events collection is empty.")
```

Comment:
Why raising an exception if the DataFrame is empty ?
Don't we only want to ignore them ?

Reply:
This filter is silly. I'll remove it.